packages/core/plugin/src/withSentryIOS.ts

```diff
@@ -79,3 +85,58 @@ export function addSentryWithBundledScriptsToBundleShellScript(script: string):
     (match: string) => `/bin/sh ${SENTRY_REACT_NATIVE_XCODE_PATH} ${match}`,
   );
 }
+
+export function modifyAppDelegate(config: ExpoConfig): ExpoConfig {
+  return withAppDelegate(config, async config => {
+    if (!config.modResults || !config.modResults.path) {
+      warnOnce('Skipping AppDelegate modification because the file does not exist.');
+      return config;
+    }
+
+    const fileName = path.basename(config.modResults.path);
+
+    if (config.modResults.language === 'swift') {
+      if (config.modResults.contents.includes('RNSentrySDK.start()')) {
+        warnOnce(`Your '${fileName}' already contains 'RNSentrySDK.start()'.`);
+        return config;
+      }
+      if (!config.modResults.contents.includes('import RNSentry')) {
```

Comment:
Let's add the imports after the start call is added.

Reply:
Updated with 0ffd26c8f16ee2e1a1bb04bca13be7a95ff2726b